kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java

```diff
@@ -155,7 +182,7 @@ int initArgs(String[] args) {
             DynMethods.builder("defaultBeelineConnect")
                 .hiddenImpl(BeeLine.class, CommandLine.class)
                 .buildChecked(this)
-                .invoke(beelineParser, cl);
+                .invoke(cl);
```

Comment:
cc @bowenliang123

Reply:
It seems my fault in #4879 . Apologize for adding the unrelated argument in the first place.

Reply:
could you help check why the GA pass and did not detect this issue?

It helps a lot to prevent it happens again.

Reply:
#4969 is trying to run Junit tests along with ScalaTest. Let's see whether it triggers the failures in beeline.